version.go

```diff
@@ -0,0 +1,3 @@
+package scalingo
+
+var Version = "3.0.4"
```

Comment:
How could we ensure that this will always be updated ?

Reply:
Just like on the CLI, we don't :sweat_smile:

Reply:
At least having something on the readme ?